src/assignment/components/AssignmentDeadline.tsx

```diff
@@ -42,7 +42,7 @@ const AssignmentDeadline: FC<AssignmentDeadlineProps> = ({ deadline }) => {
 			<DeadlineIndicator className="u-spacer-right-s" color={config[0]} shape={config[1]} />
 
 			<FlexItem shrink={false}>
-				{formatCustomTimestamp(deadline, 'DD MMMM YYYY HH:mm')}
+				{formatCustomTimestamp(deadline, 'DD-MM-YYYY, HH:mm')}
```

Comment:
comma is indeed shown in the designs
but the acceptance criteria do not include it
acceptance criteria are leading
and we don't use the comma anywhere else in avo, so its probably better to be consistent
```suggestion
				{formatCustomTimestamp(deadline, 'DD-MM-YYYY HH:mm')}
```